open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/controller/FeatureActionsController.java

```diff
@@ -79,7 +73,7 @@
     private final Map<String, DataTypeStyleConfig> myTypeKeysAndStyles = New.map();
 
     /** The style registry listener. */
-    private final VisualizationStyleRegistryChangeListener myStyleRegistryListener;
+//    private final VisualizationStyleRegistryChangeListener myStyleRegistryListener;

```

Comment:
I believe we need to keep this and I don't think this is the cause of the ticket.